newsletter/11.04.2020_newsletter.md

```diff
@@ -0,0 +1,45 @@
+---
+title: November 04, 2020 Newsletter
+permalink: /newsletter11042020/
+layout: page
+exclude: true
+---
+*This is a periodic newsletter of the interesting things we’ve seen and what we are thinking about in open source policy analysis.*
+
+**A new Tax-Cruncher Joe Biden application.** A new Tax-Cruncher
+application helps you understand how Biden’s proposed tax policies would
+influence your own financial circumstances. Kyle Pomerleau (AEI) and
+Peter Metz (AEI) created it.
+[<u>Link</u>](https://compute.studio/AEIEconomics/Tax-Cruncher-Biden/)
+
+**The New York Times releases COVID-19 data for university campuses.**
+The New York Times has open-sourced data on coronavirus cases associated
+with colleges and universities around the United States.
+[<u>Link</u>](https://github.com/nytimes/covid-19-data/tree/master/colleges)
+
+**Open-source software for renewable energy.** Led by the US Department
+of Energy, the ExaWind software library houses tools for wind turbine
+and wind power plant performance simulations. Leveraging one of the
+world’s largest supercomputers, the software helps engineers design and
+install efficient onshore and offshore wind turbines by simulating
+complex wind power plant flow dynamics.
+[<u>Link</u>](https://github.com/Exawind)
+
+**Policy Simulation Library (PSL) launches “Demo Days” series.** In the
+first PSL Demo Day, Matt Jensen (AEI) demonstrated how to specify a tax
+policy reform with PSL projects.
+[<u>Link</u>](https://www.youtube.com/watch?v=AFJq4ZrwEqA&t=1003s)
```

Comment:
This link jumps you to the middle of the video. You should be able to delete the "t=1003s" at the end

Reply:
Does [this link](https://youtu.be/AFJq4ZrwEqA) bring you to the start of the video?

Reply:
Yup that's good